src/components/techSwitcher.tsx

```diff
@@ -122,7 +125,7 @@ const TechnologySwitch = ({ type, onChangeTech, technologies, defaultTech }: Tec
 
 const Container = styled.div`
   margin: 16px 1rem 0 0;
-  width: 170px;
+  width: 185px;
```

Comment:
Can we undo this now that the tech switcher is not used any more? This changes the layout on _all_ pages that use the tech switcher.

Reply:
Does this mean that all files adding the new 'Relational' option should be removed? I quite like having 'Relational DBs' as another option for the switcher, that might be useful, but it's true that it should probably be in a separate PR.

Reminder to myself to re-add 180px to the Cockroach connector though, as we also needed to widen the box there and I took it out to save the merge conflict.

Also @millsp do we still need the changes in src/components/customMdx/index.tsx ?

Reply:
No those changes were just needed to handle html headings.